src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/OptionsTests.cs

```diff
@@ -1253,7 +1253,7 @@ public static void DefaultSerializerOptions_General()
         [Fact]
         public static void PredefinedSerializerOptions_Web()
         {
-            var options = new JsonSerializerOptions(JsonSerializerDefaults.Web);
+            var options = JsonSerializerOptions.Web;
```

Comment:
This test is checking the behavior of the constructor and shouldn't be changed. Consider adding a separate test for the singleton instead.